pallet/account-migration/src/lib.rs

```diff
@@ -456,7 +456,13 @@ pub mod pallet {
 					);
 				}
 
+				let now = <frame_system::Pallet<T>>::block_number();
+
+				l.unstaking_ring.retain(|(_, t)| t > &now);
```

Comment:
wait, where do the removed unstaking_ring go? not in free right?

Reply:
OK, so free includes staked and unstaking, deposits.

Reply:
The unstaking part belong to the user now. It'll transfer to the `staking_pot` account later.